lib/livebook/teams/requests.ex

```diff
@@ -207,6 +206,26 @@ defmodule Livebook.Teams.Requests do
     delete("/api/v1/org/deployment-groups", %{id: deployment_group.id}, team)
   end
 
+  @doc """
+  Send a request to Livebook Team API to create an agent key.
+  """
+  @spec create_agent_key(Team.t(), DeploymentGroup.t()) ::
+          {:ok, map()} | {:error, map() | String.t()} | {:transport_error, String.t()}
+  def create_agent_key(team, deployment_group) do
+    params = %{deployment_group_id: String.to_integer(deployment_group.id)}
```

Comment:
```suggestion
    params = %{deployment_group_id: deployment_group.id}
```

Reply:
I think we should make teams expect a string. Teams already converts to string for the proto message. So integer is internal, string is external. We already pass string ids in other places in this module :)